// react-spinners-css/dist/Orbitals/index.js
import _defineProperty from "@babel/runtime/helpers/esm/defineProperty";

function ownKeys(object, enumerableOnly) { var keys = Object.keys(object); if (Object.getOwnPropertySymbols) { var symbols = Object.getOwnPropertySymbols(object); if (enumerableOnly) symbols = symbols.filter(function (sym) { return Object.getOwnPropertyDescriptor(object, sym).enumerable; }); keys.push.apply(keys, symbols); } return keys; }

function _objectSpread(target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i] != null ? arguments[i] : {}; if (i % 2) { ownKeys(source, true).forEach(function (key) { _defineProperty(target, key, source[key]); }); } else if (Object.getOwnPropertyDescriptors) { Object.defineProperties(target, Object.getOwnPropertyDescriptors(source)); } else { ownKeys(source).forEach(function (key) { Object.defineProperty(target, key, Object.getOwnPropertyDescriptor(source, key)); }); } } return target; }

import React from 'react';
import './style.css';
export default function Orbitals(_ref) {
  var color = _ref.color,
      className = _ref.className,
      style = _ref.style;
  return React.createElement("div", {
    className: "lds-orbitals ".concat(className),
    style: _objectSpread({}, style)
  }, React.createElement("div", {
    className: "center",
    style: {
      background: color
    }
  }), React.createElement("div", {
    className: "inner-spin"
  }, React.createElement("div", {
    className: "inner-arc inner-arc_start-a",
    style: {
      borderColor: color
    }
  }), React.createElement("div", {
    className: "inner-arc inner-arc_end-a",
    style: {
      borderColor: color
    }
  }), React.createElement("div", {
    className: "inner-arc inner-arc_start-b",
    style: {
      borderColor: color
    }
  }), React.createElement("div", {
    className: "inner-arc inner-arc_end-b",
    style: {
      borderColor: color
    }
  }), React.createElement("div", {
    className: "inner-moon-a",
    style: {
      background: color
    }
  }), React.createElement("div", {
    className: "inner-moon-b",
    style: {
      background: color
    }
  })), React.createElement("div", {
    className: "outer-spin"
  }, React.createElement("div", {
    className: "outer-arc outer-arc_start-a",
    style: {
      borderColor: color
    }
  }), React.createElement("div", {
    className: "outer-arc outer-arc_end-a",
    style: {
      borderColor: color
    }
  }), React.createElement("div", {
    className: "outer-arc outer-arc_start-b",
    style: {
      borderColor: color
    }
  }), React.createElement("div", {
    className: "outer-arc outer-arc_end-b",
    style: {
      borderColor: color
    }
  }), React.createElement("div", {
    className: "outer-moon-a",
    style: {
      background: color
    }
  }), React.createElement("div", {
    className: "outer-moon-b",
    style: {
      background: color
    }
  })));
}
Orbitals.defaultProps = {
  color: '#7f58af',
  className: '',
  style: {}
};
